core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/util/CamelCatalogUtils.java

```diff
@@ -152,17 +152,19 @@
 			PURE_FIS_CAMEL_VERSIONS.put(FIS_20_R1_CAMEL_VERSION, "2.2.170.redhat-000013");
 		}
 
-		try {
-			Properties versionMapping = new Properties();
-			URL url = new URL(TEST_VERSIONS_URL);
-			versionMapping.load(url.openStream());
-		
-			for(String camelVersion : versionMapping.stringPropertyNames()) {
-				TEST_CAMEL_VERSIONS.add(camelVersion);
+		String camelVersionsForTesting = System.getProperty(KEY_CAMEL_TEST_VERSIONS, "").trim();
+		if (camelVersionsForTesting.equalsIgnoreCase("null")) {
```

Comment:
It does only happen in the build. I am not fully sure why but I tracked it down that the value is set to String "null" instead of null. Guess it has to do with the empty property in the pom.